src/napari_imagej/java.py

```diff
@@ -19,15 +19,14 @@
 """
 import os
 import sys
-from functools import lru_cache
 from multiprocessing.pool import AsyncResult, ThreadPool
-from typing import Any, Callable, Dict
+from typing import Callable
 
 import imagej
-import yaml
 from jpype import JClass
 from scyjava import config, jimport
 
+from napari_imagej.settings import preferences
```

Comment:
The terms "settings" and "preferences" are synonyms ([1](https://ux.stackexchange.com/questions/17442/options-preferences-settings-configurations-what-to-call-them-in-my-applicat)). Is it necessary for us to use both terms? Can we not just choose one?

Reply:
Hmm. Maybe we should move the `preferences` object to `napari_imagej.__init__.py` and rename it `settings`.

Reply:
This has been done as of 6ac06fa